src/Organizations/OrganizationForm/OrganizationContactInfoForm/ContactInfoFormGroup/EmailAddresses.js

```diff
@@ -52,17 +51,17 @@ const EmailAddresses = ({ defaultLanguage, dispatchChange, dropdownVendorCategor
             xs={12}
             md={3}
           >
-            <FieldAutoSuggest
+            <FieldAutoSuggestFinal
               items={emails}
               labelId="ui-organizations.contactInfo.emailAddress"
               name={`${name}.${valueKey}`}
               required
-              validate={[validateRequired]}
+              validate={validateRequired}
               valueKey={valueKey}
-              onSelect={(item) => {
-                fields.remove(index);
-                fields.insert(index, item);
-              }}
+              // eslint-disable-next-line no-unused-vars
+              onSelect={({ isPrimary, ...restItem }) => fields.update(index, restItem)}
+              validateFields={[]}
+              withFinalForm
```

Comment:
do we need this prop?

Reply:
Yes, moved it to [`stripes-acq-components`](https://github.com/folio-org/stripes-acq-components/pull/369)